okapi-operation-macro/Cargo.toml

```diff
@@ -15,14 +15,15 @@ proc-macro = true
 
 [dependencies]
 darling = "0.20"
-lazy_static = "1"
+lazy_static = { version = "1", optional = true}
 proc-macro2 = "1"
 quote = "1"
 syn = { version = "2", features = ["full"] }
 thiserror = "1"
 
 [features]
 axum = []
+legacy_lazy = ["lazy_static"]
```

Comment:
I don't see any reason to keep it if switching to LazyLock works. The only reason is to keep compatibility with Rust versions prior to 1.80 and IMO it is unnecessary, so I would like to drop this feature and just keep LazyLock version.